src/mono/browser/runtime/cwraps.ts

```diff
@@ -325,8 +325,10 @@ export function init_c_exports (): void {
         if (lazyOrSkip === true || maybeSkip) {
             // lazy init on first run
             wf[name] = function (...args: any[]) {
-                const isNotSkipped = !maybeSkip || !lazyOrSkip();
-                mono_assert(isNotSkipped, () => `cwrap ${name} should not be called when binding was skipped`);
+                if (!runtimeHelpers.emscriptenBuildOptions.enableAotProfiler) {
```

Comment:
`mono_assert(isNotSkipped, () => `cwrap ${name} should not be called when binding was skipped`);`

Reply:
what was the value of `name` ?

Reply:
Only for `name = mono_wasm_profiler_init_aot`

because:
`isNotSkipped: false, maybeSkip: true, lazyOrSkip: ()=>!ot.emscriptenBuildOptions.enableBrowserProfiler`

call stack:

```
onRuntimeInitializedAsync() ->
start_runtime(); -> 
mono_wasm_init_aot_profiler(runtimeHelpers.config.aotProfilerOptions); ->
cwraps.mono_wasm_profiler_init_aot(arg); -> 
mono_assert()
```

Reply:
when the first parameter of the row in the definition (`lazyOrSkip`) is a function, that function is telling if we have to bind it immediately. In such case, the subsequent call to the function should not try to bind it again, as if it was lazy. That's what the assert means. Probably :)

Reply:
Then shouldn't we just skip the functions that evaluate to false, instead of asserting?

Reply:
it should have been bound already and we should not end up in the lazy-bind path. That's what the assert was for

Reply:
Aha, we're missing `<WasmProfilers>browser;</WasmProfilers>` in the sample. Thanks for support 😄

Reply:
there is also another problem `mono_wasm_profiler_init_browser` 

it should be 
`[() => !runtimeHelpers.emscriptenBuildOptions.enableBrowserProfiler, "mono_wasm_profiler_init_browser", "void", ["string"]],`

Reply:
Right, working without editions to the sample.